src/components/home/ProjectSection/ProjectSection.tsx

```diff
@@ -2,37 +2,28 @@ import React, {Component} from 'react';
 import Project from './Project';
 import "./ProjectSection.css";
 
-const projectData: {name: string, desc: string, imageUrl: string, projectUrl: string, projectId: string, bodyId: string, imgId: string}[] = [
+const projectData: {name: string, desc: string, projectUrl: string, projectId: string}[] = [
     {
         "name": "Hate Crime Tracker", 
         "desc": `Anti-Asian hate crimes are happening everyday and we’ve built a tool to help track them. 
                 1 Thing’s Hate Crime Tracker scrapes incident reports from national media news outlets and visualizes 
                 the data to raise awareness.`,
-        "imageUrl": require("assets/home-page/placeholder-image.png"),
         "projectUrl": "/",
-        "projectId": "project-top",
-        "bodyId": "project-body-top",
-        "imgId": "project-image-top"
+        "projectId": "top"
```

Comment:
What about we call it projectId='tracker'

Reply:
Changes have been made, please check